### ISSUE TITLE
[$15][MSFT-134] [Challenge Page] TCO Logo won't have meaningful content for the screen reader

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2830]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges/30095032
2. Switch on the screen reader
3. Tap on the page header and press tab button until it reaches to the TCO19 logo
2. Listen

**Expected Result**
It should be read-out by the screen reader. Should set the alt text as TCO19

**Actual Result**
TCO Logo won't have meaningful content for the screen reader

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 3. Understandable
**WCAG Level**: AAA
**WCAG Success Criterion**: 3.1.3 Unusual Words
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=6aa53cde%2D1eef%2D4bae%2Dbfe5%2Df5f5f79e9df4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F44

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096176 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096176 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096176 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097183 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097183 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by lakshmiathreya** [comment]
We will need to test this on Prod directly - @sushilshinde @nkumar-topcoder @Glenn - perhaps a we should have a second code review and merge to Prod if second review is fine - since QA on Dev is not possible. fyi @nithyaasworld @drasticdpk @tosha5252
Same issue as in #2819 fyi @tosha5252 @drasticdpk @nithyaasworld

**Comment by drasticdpk** [comment]
Verified on Windows/Chrome/JAWS

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097183<br/><br/>```This is an automated message for crazyk via Topcoder X```
